Cargo.toml

```diff
@@ -285,7 +284,6 @@ members = [
   "components/file_system",
   "components/collections",
   "components/coprocessor_plugin_api",
-  "components/test_coprocessor_plugin/example_plugin"
```

Comment:
Why not put in the same workspace?

Reply:
same as the comment above: it causes segment fault

Reply:
Why? Seem like a bug as same projects inside one workspace should not affect on each other excepts dependency resolution.

Reply:
Seems that it's somehow linked into the test binary...

Reply:
No, it's not linked to the usual test binary.
If you exclude the `example_plugin` from the `./script/test` script, `make test` will not run the malformed test binary.

Reply:
I've made the allocator aware of testing.